src/app/partials/dasheditor.html

```diff
@@ -144,9 +144,30 @@
   <!-- Solr tab -->
   <div ng-show="editor.index == 3">
     <div class="row-fluid">
-      <h4>Solr Settings</h4>
+      <h4>Server Settings</h4>
     </div>
     <div class="row-fluid">
+      <h6>Choose the server</h6>
```

Comment:
I would have put the solr server option on the left, as it will probably be the default (and most used) ?